<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kniffel</class>
 <widget class="QWidget" name="Kniffel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kniffel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="player_input">
        <property name="placeholderText">
         <string>Namen eingeben</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="player_add_button">
        <property name="text">
         <string>Spieler hinzuf√ºgen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="player_list"/>
      </item>
      <item>
       <widget class="QPushButton" name="player_remove_button">
        <property name="text">
         <string>Spieler entfernen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="start_game_button">
     <property name="text">
      <string>Spiel starten</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
